### PR TITLE
fix: fix aws api gateway v2 event source cookies handling

### PR DESCRIPTION
**Pull request by colonist4**
*Issue #, if available:*
Issue #388

*Description of changes:*
**Change AWS API Gateway v2 event source**
- Insert Array check logic of 'set-cookie' headers
- Delete headers['set-cookie']

# Checklist

- [x] Tests have been added and are passing -> where should i insert test logic for api gateway v2 event source ?
- [ ] Documentation has been updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/event-sources/aws/api-gateway-v2.js [hunk 1]
```diff
@@ -63,7 +63,8 @@ function getResponseToApiGateway ({
   }
 
   if (headers['set-cookie']) {
-    responseToApiGateway.cookies = headers['set-cookie']
+    responseToApiGateway.cookies = Array.isArray(headers['set-cookie']) ? headers['set-cookie'] : [headers['set-cookie']]
+    delete headers['set-cookie']
```
Comment (WarFox) [inline src/event-sources/aws/api-gateway-v2.js]: may be we can remove some repeated code

```javascript
if(cookies = headers['set-cookie']) {
    responseToApiGateway.cookies = Array.isArray(cookies) ? cookies: [cookies]
    delete headers['set-cookie']
}
```

Comment (colonist4) [reply src/event-sources/aws/api-gateway-v2.js]: Thank you for your reviewing.
I refactored and push it. 
Can you check it again? thank you.

Comment (WarFox) [reply src/event-sources/aws/api-gateway-v2.js]: thank you, looks good to me

Comment (WarFox) [reply src/event-sources/aws/api-gateway-v2.js]: Please fix the failing tests

**Review by brett-vendia — APPROVED** [review]

**Pull request merged**